ns/conf/pdns.conf.var

```diff
@@ -1,6 +1,7 @@
 loglevel=5
 api=yes
 api-key=${DESEC_NS_APIKEY}
+cache-ttl=60  # as advised in https://doc.powerdns.com/authoritative/performance.html
```

Comment:
The referenced page says

> Not all backends may benefit from the packet cache. If your backend is memory based and does not lead to context switches, the packet cache may actually hurt performance.

Is that applicable to our situation?

Reply:
Good point! In fact, our backend is in-memory and has [practically no context switches](http://www.lmdb.tech/bench/inmem/scaling.html).

I pushed a commit disabling the cache. -- TODO fix base commit's title

Reply:
done